addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -446,6 +447,17 @@ var SnippetEditor = Widget.extend({
     // Private
     //--------------------------------------------------------------------------
 
+    /**
+     * @private
+     * @param {DOMElement} element
+     */
+    _hasArea: function (element) {
+        const rect = element.getBoundingClientRect();
+        return (
+             rect.bottom - rect.top > 0 &&
+             rect.right - rect.left > 0
+        );
+    },
```

Comment:
`.is(':visible')` above already does that, why is this necessary? cc @bvr-odoo

Reply:
@qsm-odoo It's for when an element size is 0px * 0px (like the logo image with some header templates). `.is(':visible')` is not enough to hide the overlay in this case

Reply:
https://api.jquery.com/visible-selector/
> Elements are considered visible if they consume space in the document. Visible elements have a width or height that is greater than zero.

Reply:
@qsm-odoo Indeed but Im testing the code and `.is(':visible')` return 'true' even if the img of the logo is 0px X 0px.
I am investigating

Reply:
@qsm-odoo I dont clearly understand why but it's always visible. I changed opacity, remove img src, etc. It doesnt change..